EdgeAgentSDK/EdgeAgentSDK/Sources/EdgeAgentSDK.docc/PrismAgent/PrismAgentHeader.md

```diff
@@ -0,0 +1,19 @@
+# EdgeAgent
+
+Prism Agent is a powerful and flexible library for working with decentralized identifiers on the Prism platform. Whether you are building a decentralized application that requires secure and private identity management, or you are working with a more traditional system that requires reliable and resilient communication between DIDs, Prism Agent provides the tools and features you need to get the job done with ease.
```

Comment:
Should the naming of `Prism Agent` should be changed to `Edge Agent`?

Reply:
Yes. Done. ;)